### PR TITLE
Cleanup custom ci to build compose app targets

### PR DESCRIPTION
**Pull request by kprosise**
Basic cleanup and readability changes. Of note:
* fixed numbering
* example for user GitHub URL changed, replacing "foundriesio" with "<your account>"
* broken reference to top of page removed, not needed.
* added `ci:read-update` to list of required token scopes.

QA Steps: Ran linter, linkcheck, and checked rendered output in browser.

No related tasks, quick fix/cleanup.

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for b2e1a2ac2c98a199c333b1aa73060ee98309ceaf are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2702/docs/artifacts/html/index.html

File: source/user-guide/custom-ci/custom-ci-for-apps.rst [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
 - builds a `FoundriesFactory Compose App`_ and pushes it to the `FoundriesFactory Registry`_;
-- composes `TUF Targets role metadata`_ compliant with FoundriesFactory TUF requirements;
+- composes `TUF Targets role metadata`_ compliant with our TUF requirements;
 - adds the composed TUF Targets to `FoundriesFactory Targets`_.
 
 Prerequisites
 -------------
 
-#. A successful build with a corresponding Target and tag, and the hardware ID, to use for this guide.
+*  A successful build with a corresponding Target and tag, and the hardware ID of a device for following along
```
Comment (vanmaegima) [inline source/user-guide/custom-ci/custom-ci-for-apps.rst:28]: I was a bit confused with this step when following the guide, maybe we should say something like:
* A successful platform build for the corresponding tag (custom-ci-devel)

File: source/user-guide/custom-ci/custom-ci-for-apps.rst [hunk 2]
```diff
@@ -45,45 +43,45 @@ Below is an example of how the prerequisites would look like:
...
 
 Creating And Setting the Access Token
 -------------------------------------
 
-The GitHub action needs to be authentication for the FoundriesFactory OTA service and the `FoundriesFactory Registry`_.
-You can create the token to add to the GitHub action via the `FoundriesFactory WebApp`_.
-The token must have ``containers:read-update`` and ``targets:read-update`` scopes to access the registry and the OTA service, respectively.
+First, you will need to authenticate the GitHub action for the OTA service and the `FoundriesFactory Registry`_.
+Create the token to add to the GitHub action via the `FoundriesFactory WebApp`_.
+The token must have ``containers:read-update``, ``targets:read-update`` and ``ci:read-update`` scopes to access the registry and the OTA service.
```
Comment (vanmaegima) [inline source/user-guide/custom-ci/custom-ci-for-apps.rst:53]:
```suggestion
The token must have ``containers:read-update``, ``targets:read-update`` and ``ci:read-update`` scopes to access the registry, FoundriesFactory CI and the OTA service.
```

**Review by vanmaegima — APPROVED** [review]

File: source/user-guide/custom-ci/custom-ci-for-apps.rst (hunk 1 above)
Comment (kprosise) [inline source/user-guide/custom-ci/custom-ci-for-apps.rst:28]:
```suggestion
*  A successful platform build for the corresponding tag ( ``custom-ci-devel`` in the example), and the hardware ID of a device for following along
```

**Comment by doanac** [comment]
Docs for e882da734e9cc75751c58ab5100bb7ab276c7ac2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2703/docs/artifacts/html/index.html

File: source/user-guide/custom-ci/custom-ci-for-apps.rst [hunk 3]
```diff
@@ -25,7 +25,7 @@ This section guides you through the steps of creating a custom CI pipeline in Gi
 Prerequisites
 -------------
 
-*  A successful build with a corresponding Target and tag, and the hardware ID of a device for following along
+*  A successful platform build for the corresponding tag ( ``custom-ci-devel`` in the example), and the hardware ID of a device for following along
```
Comment (vanmaegima) [inline source/user-guide/custom-ci/custom-ci-for-apps.rst:28]:
```suggestion
*  A successful platform build for the corresponding tag (``custom-ci-devel`` in the example), and the hardware ID (machine) of a device for following along
```

**Comment by doanac** [comment]
Docs for cd72cdabf297e18eb63b45e73a73d0362844cd1c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2704/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 5dea302d56ab74ddc3dde538e338eb0aa1942fb1 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2705/docs/artifacts/html/index.html

**Pull request merged**